### ISSUE TITLE
[wavereform] no tests (Trac #1196)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1190]
No tests.  Add some in `resources/test`.

Migrated from https://code.icecube.wisc.edu/ticket/1196
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "No tests.  Add some in `resources/test`.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[wavereform] no tests", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-08-19T18:07:37", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150826T16:36:49: 
* jbraun changed resolution from "" to "fixed"
* jbraun changed status from "new" to "closed"
* jbraun commented:

> In changeset 136899:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136899"
> Add basic testing for I3Wavereform.  Closes #1190.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
